sass-graph.js

```diff
@@ -60,10 +63,10 @@ Graph.prototype.addFile = function(filepath, parent) {
       }
     }.bind(this));
     var resolved = resolveSassPath(imports[i], _.uniq(this.loadPaths));
-    if (!resolved) return false;
+    //if (!resolved) return false;
```

Comment:
If I leave this in, not all imports for a file will be checked. Once an unresolved path has been encountered, it hits the return statement. All imports after an unresolved path will not be checked / added to the graph.